builtin/providers/aws/resource_aws_network_acl.go

```diff
@@ -213,6 +230,64 @@ func resourceAwsNetworkAclUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if d.HasChange("subnet_ids") {
+		o, n := d.GetChange("subnet_ids")
+
+		ol := o.([]interface{})
+		var pre []string
+		for _, x := range ol {
```

Comment:
Since you don't use `ol` or `nl` anywhere else, you can just do the type assertion inline in the range.